assets/js/covid19.js

```diff
@@ -209,10 +223,6 @@
         }
     }
 
-    function saveUniversity(university) {
-        localStorage.setItem('university', university);
```

Comment:
If the user came in through BB Assist we should also save the siteId as the "university" in their local storage. When they visit outside of BB Assist we can then track their requests still.